libpod/pod.go

```diff
@@ -36,6 +37,9 @@ type PodConfig struct {
 	// If true, all containers joined to the pod will use the pod cgroup as
 	// their cgroup parent, and cannot set a different cgroup parent
 	UsePodCgroup bool
+
+	// Time pod was created
+	CreatedTime time.Time `json:"createdTime"`
```

Comment:
Please use "created" or "createdat" rather than introduce another "created at time" field name.

Those names came in from images and containers.

Reply:
I don't know if I follow - this is purely internal, I don't think this should mesh with image or container fields?

Reply:
this will not be reflected in varlink ... he will define those when he wires up varlink, though the comment is valid at that time.